apps/examples/csifw_test/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_CSIFW_TEST
```

Comment:
This is not a framework. How about splitting a commit for app from framework commit?

Reply:
Done